src/Sulu/Bundle/WebsocketBundle/Admin/AppJsConfig.php

```diff
@@ -24,9 +24,10 @@ class AppJsConfig extends JsConfig
      */
     private $appManager;
 
-    public function __construct($appManager)
+    public function __construct($appManager, $bundleName, array $params)
```

Comment:
needed for JsConfig  parent constructor. Not sure why this was not set. /cc @wachterjohannes